distribution/docs/src/main/resources/content/_managing/attributes-added-by-attribute-overrides-query.adoc

```diff
@@ -0,0 +1,19 @@
+:title: Attributes Added by Attribute Overrides (Query)
```

Comment:
❓ For some reason, the font size for this section seems larger than other sections at the same level. I'm not sure if that's just a weird artifact of the way the screen cap was made or if there is some formatting issue.

Reply:
That was a screencap thing. The sections are all standardized.